packages/client/components/TimelineEventCompletedActionMeeting.tsx

```diff
@@ -32,9 +34,22 @@ const TimelineEventCompletedActionMeeting = (props: Props) => {
     endedAt,
     agendaItemCount,
     commentCount,
-    taskCount
+    taskCount,
+    locked
   } = meeting
-  const {name: teamName} = team
+  const {name: teamName, orgId} = team
+
+  const atmosphere = useAtmosphere()
+  const onUpgrade = () => {
+    SendClientSegmentEventMutation(
+      atmosphere,
+      'Timeline History Locked Meeting Upgrade CTA Clicked',
+      {
+        meetingId
+      }
+    )
```

Comment:
+1 can we use the new upgrade event names with additional properties to show the source and prompted tier?  Pseudo-code below to show my desired behavior (although I don't think `SendClientSegmentEventMutation` works this way right now)

```suggestion
    SendClientSegmentEventMutation(
      atmosphere,
      'Upgrade Intent',
      {
        source = 'Timeline History Locked Meeting Upgrade CTA',
        upgradeTier = 'pro',
        meetingId
      }
    )
```

Reply:
I've added `upgradeTier`. There is already `CTAType` of type `TierEnum`. Just double checking that `upgradeTier` is still correct.